src/components/navigation/TopMenu/TopMenu.tsx

```diff
@@ -21,6 +23,7 @@ import SearchBar, {
  */
 type Props = {
   isPlanner: boolean;
+  downloadRef: React.RefObject<HTMLDivElement | null>;
```

Comment:
I know you're aware of this already but would you mind doing the changes to make this optional?